### DISCUSSION TITLE
Search for file names in an organisation?

### DISCUSSION THREAD

**Discussion by dan2k3k4** [community/community#22292] [New to GitHub]
<p>I couldn’t find a way using the advanced search features to specifically look for file names within an organisation (over multiple repositories), or to search a specific path to a file name say:</p>
<p>“src/tests/main/components/index.js” instead of returning all index.js files.</p>
<p>Has anyone implemented an improved advanced search using the GraphQL API ? or perhaps has some gists of queries for the explorer: <a href="https://developer.github.com/v4/explorer/" rel="nofollow noopener">https://developer.github.com/v4/explorer/</a></p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/dan2k3k4">@dan2k3k4</a>! You might have better luck with getting help on this particular topic in our <a href="https://platform.github.community/" rel="nofollow noopener">Platform-specific forum</a>. I would recommend trying to open up a Topic there, as that community is specifically tuned into the GraphQL API and you might get an answer faster.</p>

**Comment by dan2k3k4** [comment]
<p><a class="mention" href="/nadiajoyce">@nadiajoyce</a> ah ok thanks, will do. I didn’t know about that forum and I only came across this forum today too.</p>

**Comment by Lokua** [comment]
<p>That link redirects to github.community homepage</p>
<blockquote>
<p><a class="mention" href="/nadiajoyce">@nadiajoyce</a> wrote:</p>
</blockquote>
<blockquote>
<p>Hi <a class="mention" href="/dan2k3k4">@dan2k3k4</a>! You might have better luck with getting help on this particular topic in our <a href="https://platform.github.community/" rel="nofollow noopener">Platform-specific forum</a>. I would recommend trying to open up a Topic there, as that community is specifically tuned into the GraphQL API and you might get an answer faster.</p>
</blockquote>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/lokua">@lokua</a> </p>
<p>At the time of that original comment’s writing, we had a separate location for the GitHub Platform Forum. Since that time, the Platform Forum has been merged into the larger GitHub Community Forum, in the form of a specific <a href="/t5/GitHub-API-Development-and/bd-p/api">API Help and Discussion board</a> (which is where that link now directs to). It is still true that that specific board would be the best place to ask any API-specific questions.</p>
<p>I hope that helps!</p>
